scripts/commit-message

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+set -e
+
+fail() {
+  (echo && sed 's/^/! /' <<<"$1" && echo) >&2
+  exit 1
+}
+
+# Find the number of the most recently opened pull request merged so far.
+# Add 1 to this to get the smallest possible number of the next pull request,
+# then add an arbitrary buffer to allow for issues and pull requests opened
+# since the creation of the most recently opened pull request merged so far.
```

Comment:
The maximum length of the branch name is a function of the number of digits in the pull request number and the number of characters in the source repository's username (e.g. `lucasschejtman`). If our search reveals…

    Merge pull request #999 from sanctuary-js/davidchambers/type-classes

we know that the next pull request number will be at least `1000`, so we expect it to be four characters in length.